resources/sdk/purecloudkotlin/config.json

```diff
@@ -0,0 +1,105 @@
+{
+    "name": "Kotlin SDK",
+    "envVars": {
+    },
+    "settings": {
+      "sdkRepo": {
+        "repo": "https://github.com/MyPureCloud/platform-client-sdk-java.git",
+        "branch": "master"
+      },
+      "versionFile": "${SDK_REPO}/version.json",
+      "logLevel": "debug",
+      "extensionsDestination": "${SDK_REPO}/build/src/main/java/com/mypurecloud/sdk/v2",
+      "swagger": {
+        "oldSwaggerPath": "${SDK_REPO}/swagger.json",
+        "newSwaggerPath": "https://api.mypurecloud.com/api/v2/docs/swagger",
+        "saveNewSwaggerPath": "${SDK_REPO}/swagger.json"
+      },
+      "swaggerCodegen": {
+        "resourceLanguage": "purecloudkotlin",
+        "codegenLanguage": "purecloudkotlin",
+        "jarPath": "${WORKSPACE}/swagger-codegen/modules/swagger-codegen-cli/target/swagger-codegen-cli.jar",
+        "configFile": "${SDK_TEMP}/config.json",
+        "extraGeneratorOptions": []
+      },
+      "releaseNoteTemplatePath": "${COMMON_ROOT}/resources/templates/githubSdkReleaseNotes.md",
+      "releaseNoteSummaryTemplatePath": "${COMMON_ROOT}/resources/templates/githubSdkReleaseNoteSummary.md",
+      "debugConfig": false,
+      "enableLoggerColor": false,
+      "mavenSettingsFilePath": "/var/build/settings.xml",
+      "mavenTarget": "deploy",
+      "apiHealthCheckUrl": "https://api.mypurecloud.com/api/v2/health/check"
+    },
+    "stageSettings": {
+      "prebuild": {
+        "preRunScripts": [
+          {
+            "type": "node",
```

Comment:
This script doesn't need to be here; it doesn't do anything. Just specify `"preRunScripts": []`

Reply:
And the same one down below.